src/Components/CreateUpdateBounty/index.js

```diff
@@ -30,49 +31,76 @@ class CreateUpdateBounty extends Component {
     render() {
```

Comment:
This component's render function has a lot of small little components that could be extracted out to their own smaller components. For example, your text area component for bounty title, bounty info, and bounty reward are all more or less the same. The only difference between all of them is just the text and labeling. Is there a way to make all these 1 component and then dynamically create this component with the text that you need and want?